rank/scripts/createIndex.ts

```diff
@@ -0,0 +1,53 @@
+import { rankClient } from '../services/elasticsearch'
+import { namespaces } from '../types/namespace'
+import { error, info } from './utils'
+
+async function go() {
+  const [indexName] = process.argv.slice(2)
+  if (!indexName) {
+    error(
+      'Please specifiy an `indexName` e.g. yarn createIndex works-with-secret-sauce'
+    )
+  }
+
+  if (!namespaces.some((n) => indexName.startsWith(n))) {
+    error(`Make sure your indexName starts with any of these - ${namespaces}`)
+  }
+
+  const indexConfig = await import(`../data/indices/${indexName}.json`)
+    .then((mod) => mod.default)
+    .catch(() =>
+      error(
+        `index config file does not exist. Create one at /data/indices/${indexName}.json} for better results`
+      )
+    )
+
+  info(`creating index ${indexName}`)
+  const { body: putIndexRes } = await rankClient.indices
+    .create({
```

Comment:
Should we reindex the prod data into these new indexes automatically? Hard to test a new mapping without any data in it...

Reply:
I didn't as you might want to do something before you do, but we can add that tomorrow.